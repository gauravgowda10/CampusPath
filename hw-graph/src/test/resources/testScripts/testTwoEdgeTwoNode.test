# Tests adding two nodes and two edges

# Create a graph
CreateGraph twoNodeGraph

# Add two nodes
AddNode twoNodeGraph x
AddNode twoNodeGraph y

# Add two edges
AddEdge twoNodeGraph x x XX
AddEdge twoNodeGraph x y XY

# Print the nodes in the graph
ListNodes twoNodeGraph
ListChildren twoNodeGraph x
ListChildren twoNodeGraph y