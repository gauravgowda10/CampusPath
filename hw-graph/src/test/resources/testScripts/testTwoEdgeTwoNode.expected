# Tests adding two nodes and two edges

# Create a graph
created graph twoNodeGraph

# Add two nodes
added node x to twoNodeGraph
added node y to twoNodeGraph

# Add two edges
added edge XX from x to x in twoNodeGraph
added edge XY from x to y in twoNodeGraph

# Print the nodes in the graph
twoNodeGraph contains: x y
the children of x in twoNodeGraph are: x(XX) y(XY)
the children of y in twoNodeGraph are: